azure-pipelines.yml

```diff
@@ -1,19 +1,60 @@
-# Starter pipeline
-# Start with a minimal pipeline that you can customize to build and deploy your code.
-# Add steps that build, run tests, deploy, and more:
-# https://aka.ms/yaml
-
 trigger:
-- master
-
-pool:
-  vmImage: ubuntu-latest
+  # start a new build for every push
+  batch: False
+  branches:
+    include:
+      - 'main'
```

Comment:
it's still called "master" on autoreject :-)

Reply:
fine from my side - I am done with all changes I wanted to do and won't interfere :-)